src/WP.php

```diff
@@ -483,15 +483,15 @@ private function wp_download_and_config( $assoc_args ) {
 			}
 		}
 
-		EE::log( 'Downloading and configuring WordPress.' );
+		\EE::log( 'Downloading and configuring WordPress.' );
 
 		$chown_command = "docker-compose exec --user=root php chown -R www-data: /var/www/";
-		EE::exec( $chown_command );
+		\EE::exec( $chown_command );
 
 		$core_download_command = "docker-compose exec --user='www-data' php wp core download --locale='$this->locale' $core_download_arguments";
 
-		if ( ! EE::exec( $core_download_command ) ) {
-			EE::error('Unable to download wp core.', false );
+		if ( ! \EE::exec( $core_download_command ) ) {
+			\EE::error('Unable to download wp core.', false );
 		}
 
 		// TODO: Look for better way to handle mysql healthcheck
```

Comment:
Do we have some issue for this in our repo? If not please create one so that it won't lost in code itself.